components/training/eventBooking.tsx

```diff
@@ -96,7 +111,11 @@ const EventCard = ({ event, count, index, eventDurationInDays, schema }) => {
                 />
               </div>
               <div className="py-0.5 text-xs uppercase text-gray-500">
-                {EventModel.TIMINGS}
+                {event.date && event.endDate
+                  ? formatTimeWithAmPm(new Date(event.date)) +
+                    " - " +
+                    formatTimeWithAmPm(new Date(event.endDate))
+                  : EventModel.TIMINGS}
```

Comment:
It can be a little confusing if you change the start time and the date doesn't change

```suggestion
                {
                event.date ? formatTimeWithAmPm(new Date(event.date) ) : '9AM' +
                " - " +
                event.endDate ? formatTimeWithAmPm(new Date(event.endDate)) : "5PM"
                }
```